tests/openwisp2/sample_monitoring/tests.py

```diff
@@ -17,7 +17,20 @@ class TestDatabase(BaseTestDatabase):
 
 
 class TestModels(BaseTestModels):
-    pass
+    def test_metric_details_field(self):
```

Comment:
Why is this test specific for sample_monitoring?

Reply:
I didn't get the query. I have added some additional fields for every model as an example for the user and @nemesisdesign suggested to have a test for testing the same. (You can check the suggestions in #102). Besides that, I am unsure if you implied something else.

Reply:
Fair enough, it just seemed odd that this is just the only test in the sample_app